src/Tests.IntegrationTests/ClassifyMessageTests.cs

```diff
@@ -19,7 +20,14 @@ public ClassifyMessageTests(IConfiguration configuration, ITestOutputHelper outp
             _configuration = configuration;
             _output = output;
             _testClient = testClients;
-            _fixture = fixture;
+
+            if (fixture == null)
+            {
+                throw new ArgumentNullException(nameof(fixture));
+            }
+
+            var useFixture = Convert.ToBoolean(_configuration["UseFixture"], CultureInfo.CurrentCulture);
```

Comment:
I'd recommend using invariant culture here to avoid any region-specific issues.
```suggestion
            var useFixture = Convert.ToBoolean(_configuration["UseFixture"], CultureInfo.InvariantCulture);
```

Reply:
Fixed in all places, thanks